src/containers/Application/ApplicationOverview.tsx

```diff
@@ -86,11 +86,11 @@ const ApplicationOverview: React.FC = () => {
   }, [elementsState, responsesLoading])
 
   const revalidateAndUpdate = async () => {
-    const revalidate = await revalidateAll(
-      elementsState as ApplicationElementStates,
-      responsesByCode as ResponsesByCode,
-      currentUser as User
-    )
+    const revalidate = await revalidateAll({
+      elementsState: elementsState as ApplicationElementStates,
+      responsesByCode: responsesByCode as ResponsesByCode,
+      currentUser: currentUser as User,
```

Comment:
I see -- it's because Typescript won't allow `elementsStateaas ApplicationElementStates`. Ugh, it's so ugly 😆